src/app/wallet/wallet/addresslookup/addresslookup.component.ts

```diff
@@ -16,11 +16,11 @@ export class AddressLookupComponent implements OnInit {
 
   log: any = Log.create('addresslookup.component');
 
-  filter: string = 'all';
+  filter: string = 'All types';
   query: string = '';
 
   public type: string = 'send';
-  public addressTypes: Array<string> = ['all', 'public', 'private'];
+  public addressTypes: Array<string> = ['All types', 'Public', 'Private'];
```

Comment:
No need to change array value, I already fixed it . 
https://github.com/particl/partgui/pull/432/files#diff-0f3b7cabc63787eee98c581387535fedR11